docs/main/ios/spm.md

```diff
@@ -59,7 +59,45 @@ Then let's sync the web app. This will add the App plugin SPM to the iOS project
 
 You can now use the App plugin normally.
 
-<em>More details coming soon</em>
+## Using SPM in an existing Capacitor Project
+
```

Comment:
I think a first section could mention the possibility of removing `ios` folder and adding it back with `npx cap add ios --packagemanager SPM` in case the user didn't do any custom changes to the project, since that's the "easiest" migration and requires no manual steps.